### PR TITLE
use ABC sorting order for tree-view items

### PR DESCRIPTION
**Pull request by just-boris**
Currently it shows all items like
* Z
* Y
* X

This neither not usable or understandable. Before we will implement fully-featured sorting, we need to fix at least in this way.

### CODE REVIEW

File: allure-generator/src/test/javascript/stubs/handlebars-compiler.js [hunk 1]
```diff
@@ -5,8 +5,6 @@ const path = require('path');
 const fs = new MemoryFS();
 const file = process.argv[2];
 
-console.log('SETUP HOHOHO');
-
```
Comment (just-boris) [inline allure-generator/src/test/javascript/stubs/handlebars-compiler.js]: This line used to fail tests. Sure, we need to get rid of this and enable JS-tests in the build again.

Comment (baev) [inline allure-generator/src/test/javascript/stubs/handlebars-compiler.js]: yep, that is my fault. I can't get them working again after merging generator & web together. Could you please take a look?

**Comment by baev** [comment]
thanks @just-boris

File: allure-generator/src/test/javascript/stubs/handlebars-compiler.js (hunk 1 above)
Comment (just-boris) [inline allure-generator/src/test/javascript/stubs/handlebars-compiler.js]: After this change, I run `yarn test` and it passed localy.
How else should I test?

**Pull request merged**